Pipfile

```diff
@@ -17,6 +17,7 @@ flask-script = "~=2.0.6"
 flask-cors = "~=3.0.9"
 confluent-kafka = "~=1.5.0"
 requests = "~=2.25.1"
+marshmallow = "~=3.10.0"
```

Comment:
```suggestion

```

Reply:
Ok, I didn't have an idea about `marshmallow` so I checked the home page [here](https://marshmallow.readthedocs.io/en/stable/index.html#) they mentioned it is an ORM so I compared it with SQLAlchemy